### PR TITLE
add git to linux dockerfile

### PR DESCRIPTION
**Pull request by jmlphd**
scipy builds are failing because git is not installed in the build container.

### CODE REVIEW

**Comment by jmlphd** [comment]
A little more context:

For the community repo, ZenCohen is running into an issue where scipy builds fail because git is not installed on the build container.

**Review by dkomisar — APPROVED** [review]

**Review by dkomisar — APPROVED** [review]

**Comment by dbast** [comment]
this gets a no from my side, as git isn't there on purpose and feedstocks have to add it as build requirement.

if community builds are different then they need additional to 1. their own github org 2. their own pinnings 3. their own compilers then also 4. their own containers (like defined here https://github.com/conda-forge/docker-images ... there are afaik more deviations that will all pop up over time).

what would be good reasons to separate everything but not the containers?

**Comment by jmlphd** [comment]
I think you're right that this is an easy change and that others will not necessarily be so.

I'm wondering if it makes sense to have the build system somehow know to which channel a flow run is publishing a package.  From there it could figure out which docker image to use to build it.  Of course that only works for linux builds.  For all other arch/OS, we'd need to have different AMIs for different agents.

At any rate, if you're okay with this change, then I'll merge once you approve.  I think if another change comes up we should have a discussion about whether we should continue to modify this dockerfile, or if we explore using a separate docker image for building community repo packages.

**Comment by dbast** [comment]
@jmlphd Is that still required?

**Pull request closed without merging**
